test/shared/utils.py

```diff
@@ -74,23 +74,38 @@ def expected_wheels(package_name, package_version, manylinux_versions=['manylinu
     # {distribution}-{version}(-{build tag})?-{python tag}-{abi tag}-{platform tag}.whl
     # {python tag} and {abi tag} are closely related to the python interpreter used to build the wheel
     # so we'll merge them below as python_abi_tag
-    python_abi_tags = ['cp27-cp27m', 'cp35-cp35m', 'cp36-cp36m', 'cp37-cp37m', 'cp38-cp38']
+    python_abi_tags = ['cp27-cp27m', 'cp35-cp35m', 'cp36-cp36m', 'cp37-cp37m', 'cp38-cp38',
+                       'pp273-pypy27_pp73', 'pp373-pypy36_pypy73']
     if platform == 'linux':
         python_abi_tags.append('cp27-cp27mu')  # python 2.7 has 2 different ABI on manylinux
-        platform_tags = []
-        for architecture in ['x86_64', 'i686']:
-            for manylinux_version in manylinux_versions:
-                platform_tags.append('{manylinux_version}_{architecture}'.format(
-                    manylinux_version=manylinux_version, architecture=architecture
-                ))
+        # We don't have PyPy 7.3.0rc1 yet on the manylinux image
+        python_abi_tags[python_abi_tags.index('pp273-pypy27_pp73')] = 'pp272-pypy_41'
+        python_abi_tags[python_abi_tags.index('pp373-pypy36_pp73')] = 'pp372-pypy3_72'
+        architectures = {'cp': ['x86_64', 'i686'], 'pp': ['x86_64']}
+        platform_tags = {}
+        for python_implemention in architectures:
+            platform_tags[python_implemention] = [
+                '{manylinux_version}_{architecture}'.format(
+                    manylinux_version=manylinux_version, architecture=architecture)
+                for architecture in architectures[python_implemention]
+                for manylinux_version in manylinux_versions
+            ]
         def get_platform_tags(python_abi_tag):
-            return platform_tags
+            return platform_tags[python_abi_tag[:2]]
     elif platform == 'windows':
         def get_platform_tags(python_abi_tag):
-            return ['win32', 'win_amd64']
+            if python_abi_tag.startswith('pp'):
+                return ['win32']
+            else:
+                return ['win32', 'win_amd64']
```

Comment:
It could be done as is done on linux now (linux is much more complicated though):
```
platform_tags = {'pp': ['win32'], 'cp': ['win32', 'win_amd64']}
def get_platform_tags(python_abi_tag):
    return platform_tags[python_abi_tag[:2]]
```

Reply:
Hmmm, myeah, why not :-)